codes/typescript/chapter_heap/my_heap.ts

```diff
@@ -4,6 +4,8 @@
  * Author: Justin (xiefahit@gmail.com)
  */
 
+import esMain from 'es-main';
```

Comment:
`import.meta.url.endsWith(process.argv[1])` may be an alternative to avoid requiring `es-main` library.